src/v2/Apps/Auctions/Routes/CurrentAuctions.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import { Box, Text, Separator } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+
+import { CurrentAuctions_currentAuctions } from "v2/__generated__/CurrentAuctions_currentAuctions.graphql"
+
+export interface CurrentAuctionsProps {
+  currentAuctions: CurrentAuctions_currentAuctions
+}
+
+const CurrentAuctions: React.FC<CurrentAuctionsProps> = ({
+  currentAuctions,
+}) => {
+  return (
+    <Box>
+      <Text variant="largeTitle">Current Auctions</Text>
+      <Separator mt={1} mb={3} />
+      {currentAuctions.edges.map(({ node }, index) => {
+        return <Box my={4} key={index}></Box>
+      })}
+    </Box>
+  )
+}
+
+export const CurrentAuctionsFragmentContainer = createFragmentContainer(
+  CurrentAuctions,
+  {
+    currentAuctions: graphql`
+      fragment CurrentAuctions_currentAuctions on SaleConnection {
+        edges {
+          node {
```

Comment:
Is this GraphQL request pulling back some form of a tree? Or do we actually have our data named thusly?

Reply:
Its both -- GraphQL lets one define an ideal "user interface" for data which is fetched from backend services. So over in  [Metaphysics](https://github.com/artsy/metaphysics) we describe what the structure of the data should look like, and then on the frontend we can request exactly what we need for the ui, and that data often looks like a tree. Some of the stuff around `edges` and `node`, etc, are conventions we've adopted for lists of items (like this, which is a list of auctions) called "connections". How it all works in practice will become clearer in time.

Reply:
Just wanted to chime in here with a link to the GQL pagination docs:

https://graphql.org/learn/pagination/

I remember when I was learning about this stuff I was really surprised about how pagination works in GraphQL - lmk if you wanna chat more about it!!

Reply:
This is super helpful @jonallured , thanks fo linking that in.